roles/kuryr/tasks/upgrade.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Delete OpenShift Kuryr pods prior to upgrade
+  shell: >
+    {{ openshift_client_binary }} get pods
+    --config={{ openshift.common.config_base }}/master/admin.kubeconfig
+    --field-selector=spec.nodeName={{ l_kubelet_node_name | lower }}
+    -o json
+    -n {{ kuryr_namespace }} |
+    {{ openshift_client_binary }} delete
+    --config={{ openshift.common.config_base }}/master/admin.kubeconfig
+    --force
+    --grace-period=0
```

Comment:
Why would you not allow it to stop gracefully?

Reply:
I based this on how openshift-sdn was handling it. Here is the reason they were forcing it https://github.com/openshift/openshift-ansible/commit/a69085bff1ac0482d7f1437ec50e2798363d17df.

But I can try without the force to see if it still works fine.